openslides/agenda/projector.py

```diff
@@ -23,15 +23,10 @@ class ItemListSlide(ProjectorElement):
 
     def get_context(self):
         pk = self.config_entry.get('id')
-        if pk is None:
-            # Root list slide.
-            context = {'tree':  self.config_entry.get('tree', False)}
-        else:
+        if pk is not None:
             # Children slide.
             if not Item.objects.filter(pk=pk).exists():
                 raise ProjectorException(_('Item does not exist.'))
-            context = {'id': pk, 'tree':  self.config_entry.get('tree', False)}
-        return context
```

Comment:
The method is namend get_FOO so I would expect a return value. Can you check that this is valid?